internal/generators/transformers/noise_timestamp.go

```diff
@@ -17,7 +18,7 @@ type NoiseTimestampLimiter struct {
 func NewNoiseTimestampLimiter(minDate, maxDate *time.Time) (*NoiseTimestampLimiter, error) {
 
 	if minDate != nil && maxDate != nil && minDate.After(*maxDate) {
-		return nil, ErrWrongLimits
+		return nil, int_utils.ErrWrongLimits
```

Comment:
Consider adding another import alias, because `int_utils` contains not only `integer` utils